.expeditor/config.yml

```diff
@@ -131,14 +137,14 @@ subscriptions:
     actions:
      - built_in:rollover_changelog
      - built_in:publish_rubygems
-     - built_in:create_github_release
-     - built_in:promote_habitat_packages
-     - bash:.expeditor/publish-release-notes.sh:
-        post_commit: true
-     - purge_packages_chef_io_fastly:{{target_channel}}/inspec/latest:
-        post_commit: true
-     - bash:.expeditor/announce-release.sh:
-        post_commit: true
+   #   - built_in:create_github_release
```

Comment:
Is it possible to skip this conditionally if pre release bump label is present ?

Reply:
I am not sure if conditional execution of actions is allowed in project_promoted workload.